dipper/models/BiolinkVocabulary.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+
+"""
+This is a class to support categorizing everything we ingest using biolink 
+categories.
+
+If you need to add an Enum for something that isn't already here,
+consult this YAML file:
+https://github.com/biolink/biolink-model/blob/master/biolink-model.yaml
+"""
+
```

Comment:
@justaddcoffee 

 __This should be an external artefact__ 

Burying this data in python code is exactly the kind of behaviour I have been moving dipper away from. 

Put it out where it can be a single source of truth for any process, 
this has nothing to do with python

Reply:
Done